test/integration.test.js

```diff
@@ -264,23 +251,15 @@ describe('registry-data (integration)', function () {
         assume(result.fingerprints.sort()).eql(BuildHeadFixture.fingerprints.sort());
         assume(result.artifacts.sort()).eql(BuildHeadFixture.artifacts.sort());
         assume(result.recommended).to.be.an('array');
-
-        result.fetchFiles(function (error, files) {
-          assume(error).is.falsey();
-          assume(files).is.an('array');
-          files.forEach(function (file) {
-            assume(file).is.an('object');
-            assume(file.url).is.a('string');
-          });
-          done();
-        });
+        done();
       });
     });
 
     it('returns a falsey value for an unknown id', function (done) {
       BuildHead.findOne({
         name: 'email',
-        env: 'proddd'
+        env: 'proddd',
+        locale: 'en-US'
```

Comment:
is that required now? was there a default before? why do we need locale here?

Reply:
@msluther because we are calling findOne, we need to provide locale since it is used as the range key after switching to dynamo (not particular to `locale` but just whatever is being used as the range key).  was getting an error about number of conditions on the keys is invalid if this range key is missing

Reply:
Do we have any callers today that don't provide a locale? I'm okay with it being required, we just need to make sure we catch callers and make sure they're providing it.

Reply:
@msluther I am not sure. I think we are migrating all the repos that we own which consumes this model so we can make sure that these provide whatever the range key that might not be previously provided? Not sure if we already have a plan for repos that we do not own that use warehouse-models, but since this is going to be published under a new major, maybe the callers can just update the findOne calls themselves when they upgrade? I can add a blurb to the README about this  (cc: @jcrugzz)

Reply:
@msluther we always provide locale when doing a findOne. Cassandra was just more flexible in that a findOne without a locale did a findAll and then just returned the first value